ruby/advanced_ruby/pattern_matching.md

```diff
@@ -592,7 +592,7 @@ end
 # => [7, 8, 9]
 ~~~
 
-As a last example we'll consider a common use case which will hopefully show where the find pattern could be a better fit than a conventional Ruby solution. It's not uncommon in Ruby to find yourself with an array of hashes or json data, and you might need to locate a record from that data. You need to match that data on a few hash keys. The data might look something like
+As a last example, we'll consider a common use case which will hopefully show where the find pattern could be a better fit than a conventional Ruby solution. It's not uncommon in Ruby to find yourself with an array of hashes or JSON data, and you might need to locate a record from that data. You need to match that data on a few hash keys. The data might look something this.
```

Comment:
The last sentence here doesn't read well. Maybe a word was missed out?

Reply:
Yeah, you're totally right, I must have missed it! Thank you for spotting that.